### PR TITLE
update default architecture and remove Nat subnet

### PR DESCRIPTION
**Pull request by outscale-hmi**
Signed-off-by: outscale_hmi <hanen.mizouni@outscale.com>

### CODE REVIEW

File: controllers/osccluster_net_controller_unit_test.go [hunk 1]
```diff
@@ -170,11 +170,11 @@ func TestCheckNetFormatParameters(t *testing.T) {
 				Network: infrastructurev1beta1.OscNetwork{
 					Net: infrastructurev1beta1.OscNet{
 						Name:    "test-net",
-						IpRange: "10.0.0.256/16",
+						IpRange: "10.0.0.256/8",
```
Comment (outscale-vbr) [inline controllers/osccluster_net_controller_unit_test.go:173]: Please change to use the new default values.

File: example/cluster-machine-template-simple.yaml [hunk 2]
```diff
@@ -39,16 +39,14 @@ spec:
     net:
       name: cluster-api-net
       clusterName: cluster-api
-      ipRange: "10.0.0.0/24"
+      ipRange: "10.0.0.0/16"
```
Comment (outscale-vbr) [inline example/cluster-machine-template-simple.yaml:42]: Please change to use the new default values.

File: example/cluster-machine-template-simple.yaml [hunk 3]
```diff
@@ -127,28 +118,28 @@ spec:
             flow: Inbound
             ipProtocol: tcp
 # IpRange to authorize access to kubernetes endpoints (kube-apiserver), you must keep it and change it with a CIDR that best suits with your environment.
-            ipRange: "10.0.0.128/26"
+            ipRange: "10.0.0.128/24"
```
Comment (outscale-vbr) [inline example/cluster-machine-template-simple.yaml:121]: Please change to use the new default values.

File: testenv/osccluster_controller_test.go [hunk 4]
```diff
@@ -1135,7 +1119,7 @@ var _ = Describe("Outscale Cluster Reconciler", func() {
 						PrivateIps: []infrastructurev1beta1.OscPrivateIpElement{
 							{
 								Name:      "cluster-api-privateip-kcp",
-								PrivateIp: "10.0.0.39",
+								PrivateIp: "10.0.4.10",
```
Comment (outscale-vbr) [inline testenv/osccluster_controller_test.go:1122]: Change file values also please

File: controllers/osccluster_net_controller_unit_test.go (hunk 1 above)
Comment (outscale-hmi) [reply controllers/osccluster_net_controller_unit_test.go:173]: It's for a use case : "check Bad Ip Range IP Net"

File: example/cluster-machine-template-simple.yaml (hunk 2 above)
Comment (outscale-hmi) [reply example/cluster-machine-template-simple.yaml:42]: The default net Ip range is 10.0.0.0/16

File: testenv/osccluster_controller_test.go (hunk 4 above)
Comment (outscale-hmi) [reply testenv/osccluster_controller_test.go:1122]: done

File: controllers/osccluster_net_controller_unit_test.go (hunk 1 above)
Comment (outscale-vbr) [reply controllers/osccluster_net_controller_unit_test.go:173]: Ok

**Comment by outscale-vbr** [comment]
Ok if you have those change ( https://github.com/outscale-vbr/cluster-api-provider-outscale/pull/78/commits/789f2855bc137a5caac2bbbd006149c88b6ca07f)

File: testenv/osccluster_controller_test.go (hunk 4 above)
Comment (outscale-vbr) [reply testenv/osccluster_controller_test.go:1122]: Ok all good

File: example/cluster-machine-template-simple.yaml (hunk 2 above)
Comment (outscale-vbr) [reply example/cluster-machine-template-simple.yaml:42]: Ok all good

**Comment by outscale-vbr** [comment]
Good jobs.

**Review by outscale-vbr — APPROVED** [review]

**Pull request merged**